src/include/simeng/arch/aarch64/Container.hh

```diff
@@ -0,0 +1,76 @@
+#pragma once
```

Comment:
I think this file would be better named `operandContainer` now. I don't see the need for a more generalised name given the single class. If this does get carried out, then maybe wait until all other reviewers approve as it'll show up as a whole new file to review on GitHub.